src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -1719,7 +1719,7 @@
   <!-- Special node, used only during nullability flow analysis -->
   <!-- PROTOTYPE(NullableReferenceTypes): Derive from BoundValuePlaceholderBase and give specific name. -->
   <Node Name="BoundValuePlaceholder" Base="BoundExpression">
-    <Field Name="Type" Type="TypeSymbol" Override="true" Null="disallow"/>
+    <Field Name="Type" Type="TypeSymbol" Override="true" Null="allow"/> <!-- We use null Type for placeholders representing out vars -->
```

Comment:
><!-- We use null Type for placeholders representing out vars --> [](start = 72, length = 64)

Is this done only in flow analysis?

Reply:
Yes, this placeholder is only created in NullableWalker.
The usage that is relevant here is in `InferMethod`.

---
In reply to: [191560002](https://github.com/dotnet/roslyn/pull/26828#discussion_r191560002) [](ancestors = 191560002)